lib/ansible/modules/cloud/vmware/vmware_host_ipv6.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Christian Kotte <christian.kotte@gmx.de>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_host_ipv6
+short_description: Enables/Disables IPv6 support for an ESXi host system
+description:
+- This module can be used to enable or disable IPv6 support for ESXi host systems in given vCenter infrastructure.
+- It also checks if the host needs to be restarted.
+version_added: 2.8
+author:
+- Christian Kotte (@ckotte) <christian.kotte@gmx.de>
+notes:
+- Tested on vSphere 6.5
+requirements:
+- python >= 2.6
+- PyVmomi
+options:
+  enabled:
```

Comment:
Can we rename this parameter to `state` in order to match other modules ?
```
present == enable
absent == disable 
```

Reply:
State absent/present means add/remove. State enabled/disabled could be an option. However, other ansible modules also use enabled true/false. Can you point me to a vmware module that uses state absent/present for enable/disable?

Reply:
There in no module in VMware space, which is absent / present for enable / disable but we can add `state` with multiple choices like `[ absent, present, enabled, disabled ]` .